### PR TITLE
node: rename `isNonEmptyDir` to `isEmptyDir`

### PR DESCRIPTION
**Pull request by dbadoy**
Simple changes for readability.

### CODE REVIEW

**Review by lightclient — COMMENTED** [review]
I find `isEmptyDir` more readable. Thank you for the contribution though!

**Comment by fjl** [comment]
@lightclient, the PR close reason you gave is weird: this PR changes it to `isEmptyDir`, which you find more readable.

**Comment by lightclient** [comment]
@fjl you're correct - I misread the change.

File: node/defaults.go [hunk 1]
```diff
@@ -104,14 +104,14 @@ func windowsAppData() string {
 	return v
 }
 
-func isNonEmptyDir(dir string) bool {
+func isEmptyDir(dir string) bool {
 	f, err := os.Open(dir)
 	if err != nil {
-		return false
+		return true
```
Comment (holiman) [inline node/defaults.go:110]: The semantics here are a bit murky. You make it return the opposite of what it returned previously, which kind of makes sense. But what if the directory does not exist?
"Is it a non-empty dir?" - "No" was the answer previously.  This makes sense. 
"Is it an empty dir?" - "Yes" is how you answer it now. Which does not make sense to me. 

What the caller needs to know is "Is this an existing and non-empty directory?". Inverting it doesn't turn it into "Is this an empty directory?", but rather "Is this either a non-existing, or an existing but empty directory?". 

So I'm sceptical. I think this change is subtly wrong, or should be named "IsNonExistOrEmpty", which is not much better than the original

Comment (dbadoy) [reply node/defaults.go:110]: ah, you right. The original is better. Thanks the review

**Review by Kantitad — COMMENTED** [review]
node/defaults.go

**Pull request closed without merging**